youtube_dl/postprocessor/embedthumbnail.py

```diff
@@ -42,8 +43,17 @@ def run(self, info):
             return [], info
 
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0', '-map', '1',
+            options = []
+            if self._crop_thumbnail:
+                options.append(
+                    '-c:a', 'copy', '-c:v', 'mjpeg',  # Copy audio stream, re-encode resulting thumb
+                    '-vf', 'crop="\'if(gt(ih,iw),iw,ih)\':\'if(gt(iw,ih),ih,iw)\'"'  # Use the crop filter on the image
```

Comment:
Wouldn't just `crop=in_h` work for youtube music , also shouldn't we replace those all appends with `+=` ,  Shh..... I wasn't here ... 😄